providers/apache/hive/pyproject.toml

```diff
@@ -65,6 +65,7 @@ dependencies = [
     "pyhive[hive_pure_sasl]>=0.7.0",
     "thrift>=0.11.0",
     "jmespath>=0.7.0",
+    "pykerberos>=1.1.13",
```

Comment:
Should it be a dependency or an optional dependency?
I don't think everyone are using kerberos?